From 90c6122daabb5606d5f77aed9a22ea1df0b57ee4 Mon Sep 17 00:00:00 2001
From: Jian Li <pyguni@gmail.com>
Date: Thu, 30 Jan 2020 23:56:09 +0900
Subject: [PATCH] [PATCH] Cherry-pick lib deps from master branch

---
 lib/BUCK      | 54 +++++++++++++++++++++++++++++++++++++++++++++++++++
 lib/deps.json |  6 ++++++
 2 files changed, 60 insertions(+)

diff --git a/lib/BUCK b/lib/BUCK
index 9e13472984..3f0d55fca2 100644
--- a/lib/BUCK
+++ b/lib/BUCK
@@ -475,6 +475,42 @@ remote_jar (
   visibility = [ 'PUBLIC' ],
 )
 
+remote_jar (
+  name = 'simpleclient',
+  out = 'simpleclient-0.5.0.jar',
+  url = 'mvn:io.prometheus:simpleclient:jar:0.5.0',
+  sha1 = 'fbbfe2300098798e3d23f93b7b14befeceacf512',
+  maven_coords = 'io.prometheus:simpleclient:0.5.0',
+  visibility = [ 'PUBLIC' ],
+)
+
+remote_jar (
+  name = 'simpleclient_common',
+  out = 'simpleclient_common-0.5.0.jar',
+  url = 'mvn:io.prometheus:simpleclient_common:jar:0.5.0',
+  sha1 = 'bfd93082d7cf85c0543c2ccc286b96c817d1090c',
+  maven_coords = 'io.prometheus:simpleclient_common:0.5.0',
+  visibility = [ 'PUBLIC' ],
+)
+
+remote_jar (
+  name = 'simpleclient_hotspot',
+  out = 'simpleclient_hotspot-0.5.0.jar',
+  url = 'mvn:io.prometheus:simpleclient_hotspot:jar:0.5.0',
+  sha1 = '0f341cb84d6713255b1ce46c7593eee50a35d414',
+  maven_coords = 'io.prometheus:simpleclient_hotspot:0.5.0',
+  visibility = [ 'PUBLIC' ],
+)
+
+remote_jar (
+  name = 'simpleclient_servlet',
+  out = 'simpleclient_servlet-0.5.0.jar',
+  url = 'mvn:io.prometheus:simpleclient_servlet:jar:0.5.0',
+  sha1 = '28d75ee90a5c2ac70eeebb0864b15885116207a4',
+  maven_coords = 'io.prometheus:simpleclient_servlet:0.5.0',
+  visibility = [ 'PUBLIC' ],
+)
+
 remote_jar (
   name = 'retrofit',
   out = 'retrofit-2.3.0.jar',
@@ -709,6 +745,24 @@ remote_jar (
   visibility = [ 'PUBLIC' ],
 )
 
+remote_jar (
+  name = 'jetty-servlet',
+  out = 'jetty-servlet-9.2.26.v20180806.jar',
+  url = 'mvn:org.eclipse.jetty:jetty-servlet:jar:9.2.26.v20180806',
+  sha1 = '7d54ae4579454d9df2c3324de7eddfbd81ffb8cc',
+  maven_coords = 'org.eclipse.jetty:jetty-servlet:9.2.26.v20180806',
+  visibility = [ 'PUBLIC' ],
+)
+
+remote_jar (
+  name = 'jetty-security',
+  out = 'jetty-security-9.2.21.v20170120.jar',
+  url = 'mvn:org.eclipse.jetty:jetty-security:jar:9.2.21.v20170120',
+  sha1 = 'ceb39cd058fa22172c6e7ef493af90891e701f23',
+  maven_coords = 'org.eclipse.jetty:jetty-security:9.2.21.v20170120',
+  visibility = [ 'PUBLIC' ],
+)
+
 remote_jar (
   name = 'jetty-continuation',
   out = 'jetty-continuation-9.2.21.v20170120.jar',
diff --git a/lib/deps.json b/lib/deps.json
index f79384f04a..a6b95f3323 100644
--- a/lib/deps.json
+++ b/lib/deps.json
@@ -130,6 +130,10 @@
     "httpclient-osgi": "mvn:org.apache.httpcomponents:httpclient-osgi:4.5.1",
     "httpcore-osgi": "mvn:org.apache.httpcomponents:httpcore-osgi:4.4.4",
     "influxdb-java": "mvn:org.influxdb:influxdb-java:2.9",
+    "simpleclient": "mvn:io.prometheus:simpleclient:0.5.0",
+    "simpleclient_common": "mvn:io.prometheus:simpleclient_common:0.5.0",
+    "simpleclient_hotspot": "mvn:io.prometheus:simpleclient_hotspot:0.5.0",
+    "simpleclient_servlet": "mvn:io.prometheus:simpleclient_servlet:0.5.0",
     "retrofit": "mvn:com.squareup.retrofit2:retrofit:2.3.0",
     "converter-moshi": "mvn:com.squareup.retrofit2:converter-moshi:2.3.0",
     "okhttp": "mvn:com.squareup.okhttp3:okhttp:3.9.1",
@@ -156,6 +160,8 @@
     "jetty-util": "mvn:org.eclipse.jetty:jetty-util:9.2.21.v20170120",
     "jetty-websocket": "mvn:org.eclipse.jetty:jetty-websocket:8.1.19.v20160209",
     "jetty-server": "mvn:org.eclipse.jetty:jetty-server:9.2.21.v20170120",
+    "jetty-servlet": "mvn:org.eclipse.jetty:jetty-servlet:9.2.26.v20180806",
+    "jetty-security": "mvn:org.eclipse.jetty:jetty-security:9.2.21.v20170120",
     "jetty-continuation": "mvn:org.eclipse.jetty:jetty-continuation:9.2.21.v20170120",
     "jetty-http": "mvn:org.eclipse.jetty:jetty-http:9.2.21.v20170120",
     "jetty-io": "mvn:org.eclipse.jetty:jetty-io:9.2.21.v20170120",
-- 
2.21.0 (Apple Git-122)

